src/log.js

```diff
@@ -2,8 +2,8 @@ const debug = require('debug');
 
 module.exports.debug = debug('horti:debug');
 module.exports.stage = debug('horti:stage');
-module.exports.info = console.log;
-module.exports.error = console.error;
+module.exports.info = console.log; //eslint-disable-line
```

Comment:
Should we consider handling `logging` the way we are currently doing it in our main codebase? i.e. using _winston library_

Reply:
IDK, I'm not massively keen on it.

For API and Sentinel I can understand that having a more performant logger is useful because the volume of logging is proportional to user engagement (bigger server = more logging), but horti only logs out when it's performing an action like upgrading, so it doesn't seem as relevant. If for some reason it takes on more responsibility we could add it later.